client/src/app/app.component.html

```diff
@@ -9,7 +9,9 @@
       <span class="icon icon-menu" (click)="menu.toggleMenu()"></span>
 
       <a class="peertube-title" [routerLink]="defaultRoute">
-        <span class="icon icon-logo"></span>
+        <object class="icon icon-logo" type="image/svg+xml" data="/client/assets/images/logo.svg">
```

Comment:
why the change to `object`? You don't motivate it.

Reply:
From CSS background-url, Webpack add a unique hash in the file name like `logo.6Z9DD98KJjk9DJ98.svg`, so right no you need to override CSS to simply customize the logo. This way we'll be able to add a query-string to handle cache in data property of the object tag and store it in the storage directory. Having a separate storage for logo and favicon allows to avoid overwriting when updating PeerTube.